tldraw/packages/core/src/lib/TLApp/TLApp.ts

```diff
@@ -173,7 +173,7 @@ export class TLApp<
         },
       },
       {
-        keys: ['delete', 'backspace'],
```

Comment:
Are you sure this will fix windows? Changing from `delete` to `del` will break the keybinding on Mac.